### DISCUSSION TITLE
CrossPage assets

### DISCUSSION THREAD

**Discussion by Valexr** [nuejs/nue#419] [Q&A]
How I can use cross page images from same folder?

`./imgs`

**Comment by tipiirai** [comment]
What do you mean by "use"?

**Comment by Valexr** [comment]
I have same images on 2 different pages

**Comment by tipiirai** [comment]
Using absolute paths? 

```md
![alt](/imgs/foo.png)
```

**Comment by Valexr** [comment]
Yep! But it’s crashes routes in GHpages 🤔

**Reply by Valexr** [reply]
One moment. Let me try…

**Reply by Valexr** [reply]
It’s working… 👌🏻

**Comment by Valexr** [comment]
I think it’s maybe useful case - shared assets folder for all site pages

**Comment by tipiirai** [comment]
There is already a "globals" folder. See: https://nuejs.org/docs/project-structure.html#globals
